### PR TITLE
Update to latest SwiftLint

### PR DESCRIPTION
**Pull request by bryanluby**
This pr updates the Swift template to use the latest version of SwiftLint (0.16.1) and fixes any new warnings it was giving off.

@vokal/ios-developers review please

### CODE REVIEW

**Review by joevandeventer — APPROVED** [review]
Looks good! Clearly I need to get better about remembering a space before my `TODO`s.

File: Vokal-Swift.xctemplate/.swiftlint.yml [hunk 1]
```diff
@@ -18,6 +18,9 @@ opt_in_rules:
 force_unwrapping:
   severity: error
 
+trailing_comma:
+  mandatory_comma: true
+
```
Comment (vokal-isaac) [inline Vokal-Swift.xctemplate/.swiftlint.yml:23]: 🎉

**Review by vokal-isaac — APPROVED** [review]
LGTM

File: Vokal-Swift.xctemplate/.swiftlint.yml (hunk 1 above)
Comment (bryanluby) [reply Vokal-Swift.xctemplate/.swiftlint.yml:23]: I knew you would like that one

**Pull request merged**